session/bootstrap.go

```diff
@@ -339,6 +339,22 @@ const (
 		WITH_GRANT_OPTION enum('N','Y') NOT NULL DEFAULT 'N',
 		PRIMARY KEY (USER,HOST,PRIV)
 	  );`
+
+	// CreatePlacementPolicyTable store the placement policys.
+	CreatePlacementPolicyTable = `CREATE TABLE IF NOT EXISTS mysql.placement_policy (
```

Comment:
Why not treat it as a virtual table like `information_schema.tables`, and only store `placementPolicy` to the metadata, just like `tableInfo`?
The disadvantage of creating a real table is that it's really complex to update the data definition between versions.